packages/react-pose/src/components/Transition/children.ts

```diff
@@ -34,22 +36,35 @@ const handleTransition = (
     enterAfterExit,
     flipMove,
     onRest,
-    ...propsForChild
+    ...propsForChilds
```

Comment:
Should really be `propsForChild` (as they get applied per child) or `propsForChildren` which is the correct pluralisation. Either's fine by me